.pre-commit-config.yaml

```diff
@@ -8,13 +8,13 @@ repos:
 
 
 - repo: https://github.com/psf/black
-  rev: 23.11.0
+  rev: 24.3.0
```

Comment:
Why you've changed this? It isn't part of your task :p

Reply:
It's pre-commit autoupdate